glossary.rst

```diff
@@ -11,11 +11,26 @@ Glossary
          A collection of binary-valued variables  (variables that can be assigned two values, for example -1, 1) 
          with associated linear and quadratic biases. Sometimes referred to in other tools as a problem.
 
+      Chain length
+         The number of qubits in a :term:`Chain`.
+
+      Chain
+         One or more nodes or qubits in a target graph that represent a single variable in
+         the source graph. See :term:`embedding`.
+
       Chimera
          The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
          connect to others via couplers, the D-Wave QPU is not fully connected.
          Instead, the qubits interconnect in an architecture known as Chimera.
 
+      Complete graph
+      Fully connected
+          See `complete graph`_. A fully connected or complete
```

Comment:
See `complete graph`_. ? Did you mean to put the description under Complete Graph and the reference under Fully connect, placed in the alphabetically-ordered place?

Reply:
That links wikipedia